test/test_secure_apis.sh

```diff
@@ -112,25 +112,6 @@ if [[ $OUT = *"\"name\": \"Another Copy Of Write below binary dir\""* ]]; then
     exit 1
 fi
 
-WRITE_BELOW_BINARY_POS=`$SCRIPTDIR/../examples/list_policies.py $PYTHON_SDC_TEST_API_TOKEN | grep -b "\"name\": \"Write below binary dir" | awk -F: '{print $1}'`
```

Comment:
Was there a specific reason to remove this section of tests?

Reply:
This test is testing the policy priorities, which AFAIK is deprecated in policy API V2.